fastlane/lib/fastlane/lane_manager_base.rb

```diff
@@ -0,0 +1,122 @@
+module Fastlane
+  class LaneManagerBase
```

Comment:
It would be 💯 to have a 1-2 sentence comment here on what class is responsible for, and what to use it for